backend/utils/db.py

```diff
@@ -87,5 +88,34 @@ def update_snack(sku: str, updates: SnackUpdateSchema) -> Snack:
         return Snack(**record)
 
 
+#Bulk Processing function
+def create_bulk_items(bulk_snacks:BulkSnackCreate)-> List[Snack]:
+    """Create a Bulk of snacks in the database"""
+    with get_db_connection() as conn:
+        cursor=conn.cursor()
+
+        snack_data=[(snack.sku,snack.name,snack.quantity) for snack in bulk_snacks]
+       # exisiting_snack="""
+        #    SELECT COUNT(*) FROM snacks WHERE sku = ?
+        #""" # Add RETURNING * 
+        print(snack_data)
+         
+        query="""            
+                        INSERT INTO snacks (sku, name, quantity)
+                        VALUES (?, ?, ?);
```

Comment:
for this pr, lets just insert with values for sku and quantity for now
```suggestion
                        INSERT INTO snacks (sku, quantity)
                        VALUES (?, ?);